openedx/openedx-platform
# SE-985 Bump completion to 2.0.0 for Ironwood

Fixes authentication issue with mobile apps.

**JIRA tickets**: [OSPR-3339](https://openedx.atlassian.net/browse/OSPR-3339)

**Discussions**: This change is already in master, see https://github.com/edx/edx-platform/pull/20039#discussion_r279601316

**Sandbox URL**: 

* LMS: https://pr20369.sandbox.opencraft.hosting/
* Studio: https://studio.pr20369.sandbox.opencraft.hosting/

(2d876fc)

**Merge deadline**: "None"

**Testing instructions**:

See https://github.com/open-craft/edx-platform/pull/165 for test instructions.

The sandbox above has already been set up for completion, mobile, and with the demo course and users. The `staff` user is a superuser.

**Author notes and concerns**:

1. Unlike https://github.com/open-craft/edx-platform/pull/165, this change does not need to upgrade `edx-drf-extensions` and `xblock`, since they are already updated.
1. https://github.com/open-craft/edx-platform/pull/165 uses completion 1.0.3, but `master` uses `2.0.0` and the [diff between 1.0.3...v2.0.0 is minimal](https://github.com/edx/completion/compare/1.0.3...v2.0.0), so we use the same here.

**Reviewers**

**Settings**
```yaml
EDXAPP_FEATURES:
    ENABLE_MOBILE_REST_API: true
```
